pkg/capabilities/v2/protoc/pkg/templates/client.go.tmpl

```diff
@@ -0,0 +1,45 @@
+package {{.GoPackageName}}
+
+import (
+    "errors"
+    "google.golang.org/protobuf/types/known/anypb"
+
+    "github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2"
+    "github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2/pb"
+
+    {{- range .Services }}
+        {{- range .Methods }}
+            {{- addImport .Input.GoIdent.GoImportPath $.GoImportPath.String }}
+            {{- addImport .Output.GoIdent.GoImportPath $.GoImportPath.String }}
+        {{- end }}
+    {{- end }}
+
+    {{- range allimports }}
+    {{.}}
+    {{- end }}
+)
+
+{{- range .Services}}
+{{ $service := . }}
+type {{.GoName}} struct {
+    // TODO config types (optional)
```

Comment:
You wanna keep those for now?

Reply:
Forgot to link the tickets, I'll do that now.